.gitignore

```diff
@@ -2,5 +2,6 @@
 .DS_Store
 .rubocop-http*
 spec/reports/*
+test/reports/*
```

Comment:
I think this is a remnant of you having an older version of `exception_handling` locally. Since the gem doesn't use `minitest` anymore, we can leave this out, and remain only ignoring `spec/reports/*`

Reply:
Removed.